frontend/src/components/post/Post.js

```diff
@@ -1,11 +1,23 @@
-import React, { useState } from 'react';
-import Like from '../like/Like'
+import React from 'react';
+import Like from '../like/Like';
+import './Post.css';
 
 const Post = ({post}) => {
   var date = new Date(post.createdDateTime);
   var formattedDate = date.toUTCString()
+
   return(
     <>
+    <div className="user-info">
+      <div className="user-profile-picture" >
+        {post.author.photo ? (
+          <img src={post.author.photo} alt="user-profile"/>
+        ) : (
+          <img src="/happy-fox.jpeg" alt="default-profile" />
+        )}
+      </div>
+      <h3>{ post.author.userName }</h3>
```

Comment:
should we give this h3 tag a class as well to be able to style it without overriding other h3's?